src/index.ts

```diff
@@ -19,29 +19,37 @@ import {
   cancel,
   showCustomizationMessage,
 } from './commands';
-import { getConfig, findFiles } from './utils/vscode-extensions';
+import { getConfig, findFiles, asRelativePath } from './utils/vscode-extensions';
 
 function initialize(context: vscode.ExtensionContext) {
   const config = getConfig().vsicons;
   const settingsManager = new SettingsManager(vscode);
   registerCommands(context);
   manageWelcomeMessage(settingsManager);
   detectProject(findFiles, config)
-    .then((path) => {
-      if (path != null) {
-        const project = fs.readFileSync(path, "utf8");
-        const projectJson = (typeof project === "string") ? JSON.parse(project) : null;
-        if (projectJson) {
+    .then((results) => {
+      if (results != null && results.length) {
+        const isInRootFolder = asRelativePath(results[0].fsPath).includes('/');
```

Comment:
This method seems to not be working. I guess that's related to my issue of having different drives. In my case, the project I'm opening for testing purposes it's located in `D:\` while the process executes in `C:\`.

Just FYI:
- `asRelativePath('c:\\GIT\\gitlab\\valudio\\phone-notify-journal\\package.json');` returns the same path.
- `asRelativePath('d:\\GIT\\gitlab\\valudio\\phone-notify-journal\\package.json');` returns `package.json`